<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStarDict::CSSEdit</class>
 <widget class="QWidget" name="QStarDict::CSSEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>203</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QTextEdit" name="m_preview">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Element</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Foreground</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="m_elementCombo">
     <property name="toolTip">
      <string>Select element</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="m_colorButton">
     <property name="statusTip">
      <string>Click to select color</string>
     </property>
     <property name="text">
      <string>#000000</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QToolButton" name="m_backgroundButton">
     <property name="statusTip">
      <string>Click to select color</string>
     </property>
     <property name="text">
      <string>#000000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFontComboBox" name="m_fontCombo">
     <property name="toolTip">
      <string>Select font</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="m_sizeSpin">
     <property name="toolTip">
      <string>Select font size</string>
     </property>
     <property name="suffix">
      <string>pt</string>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="m_boldButton">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Bold</string>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QToolButton" name="m_italicButton">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Italic</string>
     </property>
     <property name="text">
      <string>I</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QToolButton" name="m_underlineButton">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="toolTip">
      <string>Underline</string>
     </property>
     <property name="text">
      <string>U</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_elementCombo</tabstop>
  <tabstop>m_colorButton</tabstop>
  <tabstop>m_fontCombo</tabstop>
  <tabstop>m_sizeSpin</tabstop>
  <tabstop>m_boldButton</tabstop>
  <tabstop>m_italicButton</tabstop>
  <tabstop>m_underlineButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
